### DISCUSSION TITLE
The public roadmap for the development of the API

### DISCUSSION THREAD

**Discussion by drewxa** [amzn/ads-advanced-tools-docs#179] [Ideas]
Hi! Thank you for taking the time to explain the new features in such detail. I believe this information is useful to both the community and API developers.

I have an idea: If you could publish a public roadmap to guide the development of the API, that would be an excellent addition to our existing resources. This would provide a plan for all of us to follow and align our own plans with.

For instance, I want to know when the next version of Exports is coming out, which will include all remaining entities. Additionally, I'd like to know when we can expect support for the Sponsored Products keyword recommendation version 5 in Mexico and other locations.

What do you think about my idea?
Thanks

**Comment by elizafarley** [comment]
Hi @drewxa, thanks for sharing this feedback! We know this is an area of interest for developers, and I'll let you know if we have any updates to share around roadmap in the future.

In the meantime, if you (or anyone else) have other ideas around sharing roadmap, please feel free to post here!
